app/design/adminhtml/default/default/template/boltpay/system/config/button.phtml

```diff
@@ -32,7 +32,7 @@
 
         var xmlhttp = new XMLHttpRequest();
 
-        xmlhttp.open("POST", "<?=$this->getUrl('boltpay/configuration/check');?>", true);
+        xmlhttp.open("POST", "<?php echo Mage::helper('boltpay/api')->getSecureUrl('boltpay/configuration/check');?>", true);
```

Comment:
I'm actually not very familar with php. what is the difference between `<?=` and `<? echo`?

Reply:
From what I read '<?=' implies '<?php echo>'

Reply:
that's what I thought. If that's the case I'd say let's stick to  '<?=' so this diff won't contain any change other than replacing method (also '<?=' seems to be a majority across this plugin code base)